src/PresentationalComponents/CreateImageWizard/WizardStepUploadGoogle.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Form, FormGroup, TextList, TextListItem, Popover, Radio, TextContent, Text, TextInput, Title } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+
+const WizardStepUploadGoogle = (props) => {
+    const accountTypePopover = (
+        <Popover
+            hasAutoWidth
+            maxWidth='35rem'
+            headerContent={ 'Valid types of account emails' }
+            bodyContent={ <TextContent>
+                <Text>The following types of emails can have an image shared with them:</Text>
```

Comment:
Same feedback here as above. Personally I would use "The following account types ...".

Reply:
Will update this too.

Reply:
Updated in mocks as well